src/index.html

```diff
@@ -9,10 +9,59 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
```

Comment:
The href attribute should link to a CSS file, not an SCSS file. Change 'styles/index.scss' to 'styles/index.css'.